clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRecord.java

```diff
@@ -112,7 +114,7 @@ public ConsumerRecord(String topic,
                           long offset,
                           long timestamp,
                           TimestampType timestampType,
-                          long checksum,
+                          Long checksum,
```

Comment:
Adding a note here in case others are wondering: this is OK because this constructor was added during this release cycle.